packages/hasher/src/__tests__/gitFiles.test.ts

```diff
@@ -0,0 +1,59 @@
+import path from "path";
+import { EOL } from "os";
+import fs from "fs-extra";
+import tempy from "tempy";
+
+import { getListOfGitFiles } from "../gitFiles";
+
+describe("getFilesTrackedByGit()", () => {
+  it("when packageRoot is also repo root", async () => {
+    const cwd = tempy.directory();
+
+    // Creating very simple git repo
+    await fs.mkdirp(path.join(cwd, ".git"));
+    await fs.writeFile(path.join(cwd, ".gitignore"), "lib");
+    await fs.writeFile(path.join(cwd, "package.json"), "");
+    await fs.mkdirp(path.join(cwd, "lib"));
+    await fs.mkdirp(path.join(cwd, "src"));
+    await fs.writeFile(path.join(cwd, "src", "index.ts"), "");
+    await fs.writeFile(path.join(cwd, "lib", "index.js"), "");
```

Comment:
Any reason why we don't want to keep using the fixture system?

Reply:
Fixtures have their own gitignore files which mess up with these tests so we would need to manually replace/delete/move gitignore files after setting up the fixture. Which will be a bit messy, having knowledge spread in two places. Setting everything in code here prevents issues with gitignore files while still being simple to understand what is going on

Reply:
Got it. So it's just a pattern for these special test?